FogCreek/Glitch-Community
# Identify glitch team members' profiles

## Links
* https://glitch-team-ids.glitch.me/@jennschiffer or any other team member's page (that's not you if you're logged in)
* https://github.com/FogCreek/Glitch-Community-Work/issues/470
* https://docs.google.com/document/d/1hBcVZZ1XiLvkOgEglGujOcU8CkUHDfKtbyGb3QrO1Ig/edit


## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/4480480/65912873-88206480-e394-11e9-9dc8-ce16c71e55bd.png)

![image](https://user-images.githubusercontent.com/4480480/65912966-b4d47c00-e394-11e9-8cb9-8a8dfbe94d57.png)

## Changes:
* Adds a glitch icon that has a tooltip saying the person is a glitch team member, if the person is on the glitch team
* Does not add the glitch icon to the page of the logged in user, because presumably you know that you are on the glitch team already

## How To Test:
* Visit a staff member's glitch profile; visit a non-staff member's profile and make sure it doesn't show the glitch icon

## Feedback I'm looking for:
Anything!
